pyproject.toml

```diff
@@ -1,8 +1,5 @@
 [build-system]
-requires = [
-    "setuptools",
-    "wheel",
-]
+requires = ["setuptools", "setuptools_scm>=8", "wheel", "torch"]
```

Comment:
do you really need torch here? these are build-time dependencies, not runtime dependencies

Reply:
I am wondering but seems like if we use setuptools_scm it is loading the code and I get a 

```        File "<frozen importlib._bootstrap_external>", line 995, in exec_module
        File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
        File "/Users/philiploche/repos/lab-cosmo/torch-pme/src/torchpme/__init__.py", line 3, in <module>
          from . import calculators, lib, potentials, utils  # noqa
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/Users/philiploche/repos/lab-cosmo/torch-pme/src/torchpme/calculators/__init__.py", line 1, in <module>
          from .calculator import Calculator
        File "/Users/philiploche/repos/lab-cosmo/torch-pme/src/torchpme/calculators/calculator.py", line 1, in <module>
          import torch
      ModuleNotFoundError: No module named 'torch'
```

when torch is not a build deps.... I will further check.

Reply:
Lazy imports is the trick :-)

Reply:
Good question...